pandas/core/strings.py

```diff
@@ -2156,13 +2156,6 @@ def cat(self, others=None, sep=None, na_rep=None, join=None):
         `join`-keyword works as in other methods.
 
         >>> t = pd.Series(['d', 'a', 'e', 'c'], index=[3, 0, 4, 2])
-        >>> s.str.cat(t, join=None, na_rep='-')
-        0    ad
-        1    ba
-        2    -e
-        3    dc
-        dtype: object
```

Comment:
I think here it is actually useful to see the difference? (although it is deprecated?)

cc @h-vetinari

Reply:
Yes, that is true that it then should be more clear that it is deprecated, and pointing out the difference.

Reply:
@datapythonista @jorisvandenbossche 
Didn't see this on time I guess.

Indeed, the main intention of this was to show the difference, while relying on the fact that the user would see the deprecation warning if he actually used the code (but then, that was before all that docstring-validation jazz ;-)).
So, I think that example should go back in, albeit with a clearer note about the deprecation. How to avoid problems with the emitted warning? A ` # doctest: +SKIP` marker like below?